hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/RolloutManagementTest.java

```diff
@@ -83,11 +79,44 @@
 @Stories("Rollout Management")
 public class RolloutManagementTest extends AbstractJpaIntegrationTest {
 
-    @Autowired
-    private RolloutManagement rolloutManagement;
+    private static final long NOT_EXIST_IDL = 1234L;
+    private static final String NOT_EXIST_ID = "1234";
 
-    @Autowired
-    private RolloutGroupManagement rolloutGroupManagement;
+    @Test
+    @Description("Verifies that management queries react as specfied on calls for non existing entities.")
+    public void nonExistingEntityQueries() {
```

Comment:
We also have these cases in other test methods were we want to make sure that on events are send. You can add ` @ExpectEvents({ @Expect(type = RolloutGroupCreatedEvent.class, count = 0) })` for example to you test method. This will also ensure that no events at all are send.